locale/en.json

```diff
@@ -403,6 +404,16 @@
           "description": "A Tribunal Caseworker is looking at your answers and will contact you to tell you what to do next.",
           "dueDate": "This should be by <b>{{ applicationNextStep.deadline }}</b> but it might take longer than that."
         },
+        "ended": {
+          "description": "ended desc",
+          "dueDate": "ended  should be by <b>{{ applicationNextStep.deadline }}</b> but it might take longer than that.",
```

Comment:
Are you using description and dueDate? can we maybe delete these two?

Reply:
good spot, done